package.json

```diff
@@ -7,9 +7,9 @@
     "npm": ">=3.8"
   },
   "scripts": {
-    "start": "webpack-dev-server --config tools/webpack.config.babel.js --progress --env.prod=false --env.config_name=integreat",
+    "start": "webpack-dev-server --config tools/webpack.config.babel.js --progress --env.prod=false --env.config_name=integreat-debug",
```

Comment:
So the regular start command disappears. Is it not used anywhere?

Reply:
No, this is just a debug command starting the local webpack server. Also, if you want to use the live cms with the start command, you can still set the url manually back to `cms.integreat-app.de`